website/docs/r/app_configuration.html.markdown

```diff
@@ -164,7 +164,7 @@ The following arguments are supported:
 
 !> **Note:** Once Purge Protection has been enabled it's not possible to disable it. Deleting the App Configuration with Purge Protection enabled will schedule the App Configuration to be deleted (which will happen by Azure in the configured number of days).
 
-* `sku` - (Optional) The SKU name of the App Configuration. Possible values are `free` and `standard`.
+* `sku` - (Optional) The SKU name of the App Configuration. Possible values are `free` and `standard`. Defaults to `free`.
```

Comment:
https://github.com/hashicorp/terraform-provider-azurerm/blob/9df01a4577df3d9f31f4248248f7cd14ae94d426/internal/services/appconfiguration/app_configuration_resource.go#L96

Reply:
Is there an enum we can replace that with in the resource?

Reply:
Then can we raise an issue/open a PR to have it added to the swagger and leave a comment linking to the issue in the resource